src/library/rootdoc.txt

```diff
@@ -42,9 +42,9 @@ Other packages exist.  See the complete list on the right.
 Additional parts of the standard library are shipped as separate libraries. These include:
```

Comment:
Can they even still be called "parts of the standard library"? Maybe the three entries should instead be removed from this list altogether.

Reply:
This is also true, and removing is an even easier alternative

Reply:
I agree that "Additional parts of the standard library" is no longer appropriate wording, but as long as we are still shipping modules that, though separate JARs, live in the `scala._` package, I think the links are appropriate to provide, to help newcomers find their way around.

Reply:
Plus, the parallel collections are added to the list for 2.13.